spring-cloud-services-spring-connector/src/test/java/io/pivotal/spring/cloud/service/config/ConfigServerServiceConnectorIntegrationTest.java

```diff
@@ -45,7 +45,8 @@
 		classes = {
 				ConfigServerServiceConnectorIntegrationTest.TestConfig.class,
 				ConfigClientOAuth2BootstrapConfiguration.class
-		})
+		},
+		properties = "spring.cloud.config.enabled=true")
```

Comment:
For a reason I can't explain, this test will not fire up the config client without this properties. Some other thing from somewhere sets it to `false`.